C/simplicity_alloc.h

```diff
@@ -0,0 +1,34 @@
+#ifndef SIMPLICITY_SIMPLICITY_ALLOC_H
+#define SIMPLICITY_SIMPLICITY_ALLOC_H
+
+#include <stdlib.h>
+
+/* Allocate 'size' many bytes
```

Comment:
We don't need these comments on the C code.  You can have just one comment that say something like "instantiating simplicity_malloc and simplicity_free with malloc and free from the stdlib.

Reply:
I removed the comments.